.github/workflows/4_builderprecompiled_base-dev-environment.yml

```
@@ -0,0 +1,134 @@
+# This workflow downloads the source code at the given git reference
+# (branch, tag or commit), an sets up an environment (Kibana or OpenSearch)
+# to run this code and a command (build, test, ...).
+#
+# This workflow is used as a base for other workflows.
+
+name: Base workflow - Environment
```

Add `(4.x) ` prefix to the name


```suggestion
name: (4.x) Base workflow - Environment
```